src/backend/d3d12/FramebufferD3D12.h

```diff
@@ -34,10 +34,13 @@ namespace d3d12 {
                 unsigned int numRTVs = 0;
                 D3D12_CPU_DESCRIPTOR_HANDLE RTVs[kMaxColorAttachments] = {};
                 D3D12_CPU_DESCRIPTOR_HANDLE dsv = {};
+                uint32_t rtvSlots[kMaxColorAttachments] = {};
```

Comment:
nit: Seems like we prefer to use `std::array`. I also probably should have done that on line 35.